walk-in-interview/src/main/webapp/apis.js

```diff
@@ -10,4 +10,5 @@ export const API = {
   'expire-job': '/jobs/expire',
   'business-log-in': '/business-log-in',
   'applicant-log-in': '/applicant-log-in',
+  'show-job-posts-made': '/my-jobs',
```

Comment:
maybe it would be better if we renamed it to 'business-jobs-list' and then for the applicant interested list, we can call it 'applicant-interested-list' -- might help us distinguish between business and applicant specific things